images/README.md

```diff
@@ -0,0 +1,2 @@
+# Images folder
```

Comment:
this README is not needed, can you omit it from the PR?

Reply:
Doing it